assets/js/components/settings/SettingsActiveModule/Footer.js

```diff
@@ -99,7 +99,7 @@ export default function Footer( props ) {
 	} );
 
 	const { submitChanges } = useDispatch( CORE_MODULES );
-	const { clearErrors } = useDispatch( `modules/${ slug }` );
+	const { clearErrors } = useDispatch( module?.storeName );
```

Comment:
Hey @eugene-manuilov, is there any reason not to use `getModuleStoreName()` as per the IB? I would have thought you'd be in favour of this as it promotes the "single source of truth" approach...

Reply:
If we don't have the module object, then it would make sense to use that selector. But since we already have the module object, we can just read the `storeName` property. Calling that selector in this component is just an overkill.